policy-supplemental/AccessSensitiveInfoPolicy_SIAT.md

```diff
@@ -0,0 +1,13 @@
+| **Sensitive Information Access Table** 	|  	|
+|---	|---	|
```

Comment:
Move this down below the Role/Individual table line.

Reply:
Done